po/fa.po

```diff
@@ -3279,29 +3279,35 @@ msgstr ""
 
 #: src/basic-syntax/references.md:3
 msgid "Like C++, Rust has references:"
-msgstr ""
+msgstr "مانند <span dir=ltr>C++</span>، راست دارای مراجع است:"
```

Comment:
please used to [Bidirectional_text](https://en.wikipedia.org/wiki/Bidirectional_text) like [RLM](https://unicode-explorer.com/c/200F) for complex text. I think it's better than use "span dir =ltr".